CLAUDE.md

```
@@ -1,378 +1,1438 @@
+        - Texture size appropriate for web (stub acceptable)
+        - File format compatibility
+        """
+```
+
+---
+
+## 🚀 Deployment Workflow
+
+### Production URLs
+
+- **Frontend**: https://app.devskyy.app (Vercel)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)